src/binding.rs

```diff
@@ -0,0 +1,4 @@
+#![allow(non_upper_case_globals)]
```

Comment:
Add a comment here about future plans - something to the effect of 

> TODO In the immediate term, handle `cppgc::Member` due to its complex definition. There are 16 different memory representations it can have. In the short term, convert the rest of the basic size definitions for safety and memory savings. In the longer term, start migrating actual structs and functions.